bindings/capacitorjs/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@iota/wallet-mobile",
+  "version": "0.0.1",
+  "description": "Capacitor plugin binding to the wallet library",
+  "homepage": "https://github.com/iotaledger/wallet.rs#readme",
+  "main": "dist/esm/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "files": [
+    "android/src/main/",
+    "android/build.gradle",
+    "dist/",
+    "ios/Plugin/",
+    "IotaWalletMobile.podspec"
+  ],
+  "scripts": {
+    "verify": "npm run verify:ios && npm run verify:android && npm run verify:web",
+    "verify:ios": "cd ios && pod install && xcodebuild -workspace Plugin.xcworkspace -scheme Plugin && cd ..",
+    "verify:android": "cd android && ./gradlew clean build test && cd ..",
+    "build": "npm run clean && tsc && rollup -c rollup.config.js",
+    "clean": "rm -rf ./dist",
+    "watch": "tsc --watch",
+    "prepublishOnly": "npm run build",
+    "preinstall": "npm run build && add_libs.sh"
```

Comment:
you might need to add `bash add_libs.sh`, without it it doesnt work for me

Reply:
Thanks I've just pushed the fix